### DISCUSSION TITLE
Christmas lights - Is this possible?

### DISCUSSION THREAD

**Discussion by ecowitness** [merbanan/rtl_433#2212] [Q&A]
I would like to have my seasonal/holiday outdoor lighting turned on only when there is someone in the area that could view it. Leaving the lights on for hours at a time is a waste of energy. I do not live on a busy street and there are large periods of time when my displays are illuminated and there is not a single person to view them.

I am thinking that it might be possible to capture/track cell phone IMEI. The biggest hurdle for me is that I know nothing about how to do this. I don't know the frequencies. I don't know how often an IMEI is available on a frequency. There are probably many frequencies. This is probably not simple or someone would have already done this.

I thought about detecting wifi from cell phones, but then remembered that I don't turn mine on. I considered using automobile tire pressure sensors, but this isn't reliable. I thought about a video camera and detection, but then I have to deal with snow and wildlife. I even had some short lived thoughts about using a Ring doorbell. 

I am familiar with rtl_433 and currently use multiple copies of the software on the various frequencies of my IOT devices. I have brought my thoughts to this forum for others to ponder.

Might people more familiar with cell phone technology determine if this is feasible? Is anyone aware of any existing project like this?

I would like to experiment with:
- beginning around dusk until 10 p.m.
- capture any cell phone IMEI near me ((the main point of this discussion))
- check a list of already captured IMEI
- - and if not in the list
- - - add it to the list
- - - turn the lights on for 5 minutes or so and then off
- after 10 pm
- - reset the list to null for the next day

If the cost of a receiver is cheap enough, then I might consider one at each end of the block and could program an in/on and out/off automation.

If a receiver is effective, my neighbors and I could share a web of automation and we could all benefit from an energy savings. Think something like a zigbee wall switch in an outdoor box on a stake in the yard. Mesh to the next yard. Each home a cluster. We could probably do that now, but we need a trigger.

Thanks to all who read and respond.

**Comment by mhardy6351** [comment]
I dont have an answer on ttracking cell phones, but there is quite a bit of
discussion in the wled and x-lights holiday ligh forums on facebook and
github.

With the new Bluetooth low energy integrations in home assistant and
beacons, you may be a better way to trigger a show.

**Comment by zuckschwerdt** [comment]
I wouldn't pursue anything towards intercepting LTE, especially grabbing an IMEI might be illegal even. There is a wide range of frequency bands to consider and you will need an advanced SDR like LimeSDR.
I'd listen for Wifi probe requests, which needs a Wifi receiver with monitor mode, and also Bluetooth discovery requests.

But did you consider a batch of PIR sensors? Very cheap, energy efficient, mobile and well supported with rtl_433 -- you can setup zones, deduce which direction people are traveling, and later use them for home security.
